tests/regression/nginx-overrides.yaml

```
@@ -0,0 +1,178 @@
+    output:
+      status: 200
+      log:
+        no_expect_ids: [920100]
+  - rule_id: 920100
+    test_ids: [14]
+    reason: "Invalid request line produces a 400 Bad Request on Nginx. This prevents matching any rules."
+    output:
+      status: 400
+  - rule_id: 920100
+    test_ids: [16]
+    reason: "Seems like Nginx does a normalization of the request line, and if <schema>://<hostname> is sent (without a path) the request path becomes / by default"
+    output:
+      log:
+        no_expect_ids: [920100]
+  - rule_id: 920181
+    test_ids: [1]
+    reason: "Nginx returns 400 if both Content-length and Transfer-Encoding chunked are present"
+    output:
+      status: 400
+  - rule_id: 920260
+    test_ids: [1, 3]
+    reason: |
+      Nginx replaces the `%u` in the URI with `\xf`. This might be a bug.
+      See https://github.com/owasp-modsecurity/ModSecurity/issues/3135
+    output:
+      status: 200
+      log:
+        no_expect_ids: [920260]
+  - rule_id: 920270
+    test_ids: [4]
+    reason: "Header host with null byte causes Apache to error before it gets to CRS. Nginx allow this and libModSecurity correctly matches the rule"
+    output:
+      status: 200
+      log:
+        expect_ids: [920270]
+  - rule_id: 920274
+    test_ids: [1]
+    reason: "Nginx will accept the value for the Host header, the rule should trigger"
+    output:
+      status: 200
+      log:
+        expect_ids: [920274]
+  - rule_id: 920290
+    test_ids: [4]
+    reason: "Nginx returns 400 if Host header is missing"
+    output:
+      status: 400
+  - rule_id: 920360
+    test_ids: [1]
+    reason: Requires ARG_NAME_LENGTH to be set to an appropriately low value
+    output:
+      status: 200
+      log:
+        no_expect_ids: [920360]
+  - rule_id: 920370
+    test_ids: [1]
+    reason: Exceeds PCRE limits
+    output:
+      status: 200
+      log:
+        no_expect_ids: [920360]
+  - rule_id: 920380
+    test_ids: [1]
+    reason: Requires MAX_NUM_ARGS to be set to a sufficiently low value
+    output:
+      status: 200
+      log:
+        no_expect_ids: [920280]
+  - rule_id: 920390
+    test_ids: [1]
+    reason: Exceeds PCRE limits
+    output:
+      status: 200
+      log:
+        no_expect_ids: [920390]
+  - rule_id: 920430
+    test_ids: [8]
+    reason: "If the HTTP Protocol Version is invalid, Nginx take action before modsecurity sending a 505 response code."
+    output:
+      status: 505
+  - rule_id: 920610
+    test_ids: [2]
+    reason: "nginx happily accepts fragments"
+    output:
+      status: 200
+      log:
+        expect_ids: [920610]
+  - rule_id: 920620
+    test_ids: [1]
+    reason: "nginx retains multiple separate Content-Type headers, which is what this rule is looking for"
+    output:
+     log:
+      expect_ids: [920620]
+  - rule_id: 930110
+    test_ids: [7]
+    reason: "nginx does not strip the dots from the URL (which http apparently does), hence ModSecurity sees them and the rule matches"
```

nit
```suggestion
    reason: "nginx does not strip the dots from the URL (which httpd apparently does), hence ModSecurity sees them and the rule matches"
```
Just to mention it: the same behaviour of not stripping the dots that leads to matching 930110 is happening in Coraza. So maybe the override should live in httpd turning the rule to expect_ids. I will open an Issue/PR.